docs/proposals/scheduling-suspension/README.md

```diff
@@ -0,0 +1,169 @@
+
+## Motivation
+
+<!--
+当前的 Karmada 无法提供队列和优先级等能力，来将有限的资源分配给高优先级任务/负载的能力。当用户有这类需求时，他们只能手动按顺序创建
+PropagationPolicy 来优先调度。在负载和任务较少时这样没有问题，但是当一个联邦系统有大量的Job类负载时，手动无疑是效率低下和不精确的。
+
+同时不同的用户会有不同的需求，如多租户/多队列/优先级队列调度能力。当 ResourceBinding 支持暂停调度后，用户就可以去实现自己的 Controller 并
+设计自己的队列系统。
+-->
+
+**Karmada** currently lacks capabilities such as queuing and prioritization, which are necessary to allocate limited resources to high-priority tasks/workloads. When users have such needs, they can only manually create `PropagationPolicy` in sequence to prioritize scheduling. This is not a problem when there are fewer workloads and tasks, but it is inefficient and imprecise when a federated system has a large number of job-type workloads.
+
+Different users also have different requirements, such as multi-tenant, multi-queue, and priority queue scheduling capabilities. Once `ResourceBinding` supports the suspension of scheduling, users will be able to implement their own Controllers and design their own queuing systems.
+
+### Goals
+
+- Provide the capability to **pause resource scheduling**
+- Provide the capability to **resume resource scheduling**
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+<!--
+作为一名管理者，我希望 Karmada 能够允许我们通过一些方式在外部实现队列能力，从而在调度各部门的 Job 时能够限制各自的容量以及优先级，
+当资源不足时能够做到 Karmada 优先调度高优先级任务，这将帮助集群更好的利用有限的资源。
+-->
+
+As a manager, I hope that **Karmada** could allow us to implement external queue capabilities, which would enable us to limit the capacity and priority of jobs from different departments when scheduling. When resources are scarce, Karmada should prioritize higher-priority tasks. This will help the cluster make better use of limited resources.
+
+#### Story 2
+
+<!--
+作为一名用户，我希望当我创建了负载和对应的 PropagationPolicy 之后不要立刻调度，直到我取消调度暂停。
+-->
+
+As a user, I would like the scheduling not to start immediately after I have created the workload and the corresponding **PropagationPolicy**, until I lift the scheduling pause.
+
+## Design Details
+
+<!--
+通过拓展 PropagationPolicy/ClusterPropagationPolicy.Spec.Suspension，在其结构中我们引入了 SuspendScheduling 字段来表示暂停调度，
+同时此字段会传递到 ResourceBinding/ClusterResourceBinding 资源，Karmada Scheduler 组件需要根据 SuspendScheduling 来决定此时
+是否要开始调度对应的 ResourceBinding/ClusterResourceBinding。
+-->
+
+By extending `PropagationPolicy/ClusterPropagationPolicy.Spec.Suspension`, we have introduced the `SuspendScheduling` field to indicate a pause in scheduling. This field is also propagated to `ResourceBinding/ClusterResourceBinding` resources. The **Karmada Scheduler** component needs to determine whether to start scheduling the corresponding `ResourceBinding/ClusterResourceBinding` based on `SuspendScheduling`.
+
+### API Change
+
+```go
+// Suspension defines the policy for suspending different aspects of propagation.
+type Suspension struct {
+	...
+	
+    // SuspendScheduling controls whether scheduling should be suspended.
+    // +optional
+    SuspendScheduling *bool `json:"suspendScheduling,omitempty"`
```

Comment:
Maybe `Scheduling` is enough?

Reply:
thanks! do u have any more user case can share to me? i mean do u need this feat? thank u again.